### ISSUE TITLE
[SAP][Analytical Table]: Resizing the table columns can be performed using mouse but same cannot be performed using key

### ISSUE THREAD

**Issue by thisisjasmeet** [SAP/ui5-webcomponents-react#4397]
### Describe the bug

Resizing of columns is only possible using mouse.its not possible via keyboard

### Isolated Example

https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable--docs

### Reproduction steps

1.resize columns using mouse (possible)
2. try to reach resizer bar using keyboard (its not focusable)

### Expected Behaviour

user should be able to resize using keyboard only.

### Screenshots or Videos

https://user-images.githubusercontent.com/57886348/227116687-046e40ec-4bea-4759-ae32-437a14bd36ac.mov

### UI5 Web Components for React Version

1.10.3

### UI5 Web Components Version

1.10.3

### Browser

Chrome

### Operating System

macos

### Additional Context

BCP Accessibility testing issue:- 
Internal Incident: 2370007500, ACC 270.12_Level A_ Table_Resizing the table columns can be performed using mouse but same cannot be performed using key

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by Vivek-Vishal** [comment]
@Lukas742 - Any updates on this issue ?
